heidrun/TN_mods.rb

```diff
@@ -0,0 +1,149 @@
+Krikri::Mapper.define(:tn_mods, :parser => Krikri::ModsParser) do
+  provider :class => DPLA::MAP::Agent do
+    uri 'http://dp.la/api/contributor/tn'
+    label 'Tennessee Digital Library'
+  end
+
+  dataProvider :class => DPLA::MAP::Agent do
+    providedLabel record.field('mods:recordInfo', 'mods:recordContentSource')
+  end
+
+  isShownAt :class => DPLA::MAP::WebResource do
+    uri record.field('mods:location', 'mods:url')
+         	  .match_attribute(:usage, 'primary display')
+              .match_attribute(:access, 'object in context')
+  end
+
+  preview :class => DPLA::MAP::WebResource do
+    uri record.field('mods:location', 'mods:url')
+              .match_attribute(:access, 'preview')
+  end
+
+  originalRecord :class => DPLA::MAP::WebResource do
+    uri record_uri
+  end
+#they have a mapping for format and rights in the Web Resource as well...not sure how to implement
```

Comment:
Can you share an example? I'm guessing this would be for `preview`, or maybe `object`.